### ISSUE TITLE
As a CDN, *.qbox.me is blocked

### ISSUE THREAD

**Issue by l5h5t7** [EFForg/privacybadger#1340]
*.qbox.me is blocked, which should be add to yellowlist.
Sample site: https://linux.cn/

**Comment by ghostwords** [comment]
Could you run the following JavaScript code in your Privacy Badger's background page console, and send me what it prints out?

```javascript

(function () {
  const STR = "qbox.me";
  console.log("**** ACTION_MAP for", STR);
  _.each(badger.storage.getBadgerStorageObject('action_map').getItemClones(), (obj, domain) => {
    if (domain.indexOf(STR) != -1) console.log(domain, JSON.stringify(obj, null, 2));
  });
  console.log("**** SNITCH_MAP for", STR);
  _.each(badger.storage.getBadgerStorageObject('snitch_map').getItemClones(), (sites, domain) => {
    if (domain.indexOf(STR) != -1) console.log(domain, JSON.stringify(sites, null, 2));
  });
}());
```

This should print the decisions your Badger reached for all "qbox.me" domains, and also the site domains Badger saw "qbox.me" domains perform tracking on.

To open Badger's background page console in your Chrome browser, visit "chrome://extensions", make sure "Developer mode" is checked (in the upper right), find the row for Privacy Badger, and click on the "background page" link in that row. Then, in the newly appeared window, make sure the "Console" tab is selected, click somewhere within the ">" prompt, paste the above JavaScript code, and press the Enter key.

**Comment by l5h5t7** [comment]
```
VM131:3 **** ACTION_MAP for qbox.me
VM131:5 dn-acac.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492834223286,
  "userAction": ""
}
VM131:5 dn-blog-swineson-me.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492763838628,
  "userAction": ""
}
VM131:5 dn-breeswish-org.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1493082955316,
  "userAction": ""
}
VM131:5 dn-cnode.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492866989957,
  "userAction": ""
}
VM131:5 dn-coding-net-avatar.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492867041500,
  "userAction": ""
}
VM131:5 dn-coding-net-production-avatar.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492895036651,
  "userAction": ""
}
VM131:5 dn-coding-net-production-pp.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492779653302,
  "userAction": ""
}
VM131:5 dn-coding-net-production-static.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1493171151029,
  "userAction": ""
}
VM131:5 dn-discoursecn.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492811715046,
  "userAction": ""
}
VM131:5 dn-growing.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492673730945,
  "userAction": ""
}
VM131:5 dn-kdt-img-test.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1493056642315,
  "userAction": ""
}
VM131:5 dn-lbstatics.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492913877646,
  "userAction": ""
}
VM131:5 dn-leanote.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492727422057,
  "userAction": ""
}
VM131:5 dn-linuxcn.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1493098845216,
  "userAction": "user_cookieblock"
}
VM131:5 dn-mhke0kuv.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1493014019939,
  "userAction": ""
}
VM131:5 dn-noman.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492715314190,
  "userAction": ""
}
VM131:5 dn-oneapm-activity.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492931619204,
  "userAction": ""
}
VM131:5 dn-phphub.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492888008202,
  "userAction": ""
}
VM131:5 dn-pianyi.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492742575704,
  "userAction": ""
}
VM131:5 dn-qdan.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492950760604,
  "userAction": ""
}
VM131:5 dn-qs-static.qbox.me {
  "dnt": false,
  "heuristicAction": "allow",
  "nextUpdateTime": 1492817111799,
  "userAction": ""
}
VM131:5 dn-sdkcnssl.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1493048257199,
  "userAction": ""
}
VM131:5 dn-senra.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492734706595,
  "userAction": ""
}
VM131:5 dn-shimo-image.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492934369456,
  "userAction": ""
}
VM131:5 dn-sstatic.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492707233327,
  "userAction": ""
}
VM131:5 dn-staticdown.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1493055270608,
  "userAction": ""
}
VM131:5 dn-staticfile.qbox.me {
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1492873425431,
  "userAction": ""
}
VM131:5 dn-t2ipo.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492969032650,
  "userAction": ""
}
VM131:5 dn-www-swineson-me.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1493074903523,
  "userAction": ""
}
VM131:5 dn-yepster.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1493177339834,
  "userAction": ""
}
VM131:5 dn-zine-static.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492691203971,
  "userAction": ""
}
VM131:5 dn-zohead.qbox.me {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1492770748642,
  "userAction": ""
}
VM131:5 qbox.me {
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1492734391336,
  "userAction": ""
}
VM131:7 **** SNITCH_MAP for qbox.me
VM131:9 qbox.me [
  "chasedream.com",
  "wpjam.com",
  "maguang.net"
]
undefined
```

**Comment by ghostwords** [comment]
This bug should be fixed in Privacy Badger version 2017.6.13.1 (by #1403 and #1428). Please let me know if you run into it again with the new version.
